### DISCUSSION TITLE
Make for Surviv Reloaded

### DISCUSSION THREAD

**Discussion by TheMarksmanXII** [IceHacks/SurvivCheatInjector#935] [Ideas]
There has been a new surviv.io created with content from before Kong (👿) bought the game. 

Links:
https://survivreloaded.com
https://resurviv.io

Please port the injector for this website! That would be great :)

**Comment by humphreygaming** [comment]
I'm aware of resurviv.io and am glad that they are taking on that task of creating their own spin-off of sorts. Congratulations to Hasanger.

While, yes, it may be easy to port stuff over (it's open source), I don't think messing with this game is good. It's best to keep this stuff in the past, let bygones be bygones.
